libglusterfs/src/graph.c

```diff
@@ -528,13 +528,14 @@ glusterfs_graph_unknown_options(glusterfs_graph_t *graph)
     return 0;
 }
 
-static void
+static int
 fill_uuid(char *uuid, int size, struct timeval tv)
 {
     char now_str[GF_TIMESTR_SIZE];
 
     gf_time_fmt_tv(now_str, sizeof now_str, &tv, gf_timefmt_dirent);
-    snprintf(uuid, size, "%s-%d-%s", gf_gethostname(), getpid(), now_str);
+    return snprintf(uuid, size, "%s-%d-%s", gf_gethostname(), getpid(),
+                    now_str);
```

Comment:
To make it easier to check for errors in the caller, I think this could be better in this case (we don't really need to know the length of the string).
```suggestion
    ret = snprintf(uuid, size, "%s-%d-%s", gf_gethostname(), getpid(),
                   now_str);
    if (ret >= size) {
        ret = -1;
    }
    return ret;
```